test/e2e/cleanup.sh

```diff
@@ -40,12 +40,24 @@ az account set -s $SUBSCRIPTION_ID_TO_CLEANUP
 (( deadline=$(date +%s)-${expirationInSecs%.*} ))
 # find resource groups created before our deadline
 echo "Looking for resource groups created over ${EXPIRATION_IN_HOURS} hours ago..."
-for resourceGroup in $(az group list | jq --arg dl $deadline '.[] | select(.name | contains("acse-test") | not) | select(.tags.now < $dl).name' | tr -d '\"' || ""); do
+if [ -z "$RESOURCE_GROUP_SUBSTRING" ]; then
+  for resourceGroup in $(az group list | jq --arg dl $deadline '.[] | select(.name | contains("acse-test") | not) | select(.tags.now < $dl).name' | tr -d '\"' || ""); do
```

Comment:
`az --query` can probably handle this without piping to `jq` if you want to simplify.

Are we just trying not to match RGs with `acse-test` in the name? Here's the JMESPath way for example:

```
 az group list --query "[? !contains(name, 'acse-test')]"
```

Reply:
+ 1

this should be does _not_ contain though so ` az group list --query "[?!contains(name, 'acse-test')]"`

Reply:
I think the fact that we want to grab both the name *and* tags metatdata from the response suggests `jq` is the right tool, right?